binary_search/binary_search.py

```diff
@@ -0,0 +1,54 @@
+from random import randint
+
+
+def binary_search_iterative(array, left, right, element):
```

Comment:
* Need only two params here - `array` and `element`. Others can be generated.
* Function names should be `binary_search` and `binary_search_recursive`.